engine/access/storage/Blocks.go

```diff
@@ -0,0 +1,22 @@
+package storage
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+type Blocks struct {
+	storage.Blocks
+}
+
+func (blocks *Blocks) Store(block *flow.Block) error {
+	err := blocks.Blocks.Store(block)
+	return panicOnError(err)
```

Comment:
I am not a Go expert, so please take this with a grain of salt 😉 :
I think  `log.Fatal()` would be preferable to a plain old panic. 
Also, note that not each error is automatically a reason to crash the node. For example:
* If I am not mistaken, `Store` can return an `storage.ErrAlreadyExists` error (?)

Reply:
Another similar thought: I was trying to figure out which error we should expect when we're out of disk, and I can't find a sentinel for it https://pkg.go.dev/github.com/dgraph-io/badger#pkg-variables. Maybe there would be an underlying OS error we can check for -- I'm not sure exactly what it would look like once it goes through Badger. But I agree with Alex, however we do this, we should make sure we pass through any expected errors

Reply:
Also noting my suggestion from Slack yesterday here:

I suspect we would want to have this behaviour in most (all?) cases. If the disk is full, the node is in an unhealthy state and we should crash. If this is true, moving this check to a more common code-path would be simpler. One option is `update`/`insert` where all DB inserts pass through. If we don't want this behaviour for every case, we could use a lighter-touch wrapper similar to `operation.SkipDuplicates` to wrap each transaction that should cause a crash in this manner.